share/rocmcmakebuildtools/cmake/ROCMInstallTargets.cmake

```diff
@@ -131,21 +131,51 @@ function(rocm_install_targets)
         set(EXPORT_FILE ${PARSE_EXPORT})
     endif()
 
+    set(PRIVATE_LIB OFF)
```

Comment:
No need to make an intermediate variable, you can just check `if(PARSE_PRIVATE)` everywhere.

Reply:
I have made all the requested changes but I seem to be running into test failures now. Not sure if this change is causing the issue or is it something else that I am missing, but my changes are now breaking CI. I am looking into it.

Reply:
I have fixed the CI failures.